pkg/datapath/loader/netlink.go

```diff
@@ -333,6 +333,10 @@ func attachTCProgram(link netlink.Link, prog *ebpf.Program, progName, bpffsDir s
 // Direction is passed as netlink.HANDLE_MIN_{INGRESS,EGRESS} via tcDir.
 func removeTCFilters(ifName string, tcDir uint32) error {
 	link, err := netlink.LinkByName(ifName)
+	if errors.As(err, &netlink.LinkNotFoundError{}) {
```

Comment:
I think this will panic because it is not nil:
>  As panics if target is not a non-nil pointer to either a type that implements error, or to any interface type. 
https://pkg.go.dev/errors#As